### ISSUE TITLE
(Project Gallery) Figure out some way to archive older projects

### ISSUE THREAD

**Issue by ghost** [GovLab/academy-website#12]
Chris mentioned it might be useful to figure out a way to archive older projects on the project gallery page, so that we don't lose the info.

Maybe just a simple button at the bottom like "Show More Projects" that just expands on the same page (e.g. google image search)

Thoughts?

**Comment by chrswng** [comment]
Yeah, dino said ok to delete but that seems weird to me to delete all that. We should talk about it together.
